HashMapSort/src/test/java/com/AndrieievYurii/ElementsDistributionTest.java

```diff
@@ -0,0 +1,117 @@
+package com.AndrieievYurii;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+
+import static junit.framework.Assert.assertEquals;
+
+
+public class ElementsDistributionTest {
+
+    @Test
+    void shouldReturnSortedResultWhenWeGetStringInput(){
+        ElementsDistribution elementsDistribution = new ElementsDistribution();
+        HashMap<Character, Integer> expectedCharToCount = new HashMap<>();
+        expectedCharToCount.put('D',3);
+        expectedCharToCount.put('L',3);
+        expectedCharToCount.put('O',2);
+        expectedCharToCount.put('!',1);
+        expectedCharToCount.put('R',1);
+        expectedCharToCount.put('E',1);
+        expectedCharToCount.put('W',1);
+        expectedCharToCount.put('H',1);
+        elementsDistribution.consoleOutput(expectedCharToCount);
+        String input = "Hello worlddd!";
+        HashMap<Character, Integer> charToCount = elementsDistribution.elementDistribution(input);
+        elementsDistribution.consoleOutput(charToCount);
+        assertEquals(expectedCharToCount, charToCount);
+    }
+
+    @Test
+    void shouldReturnSortedResultWhenWeGetStringInputSecondCase(){
+        ElementsDistribution elementsDistribution = new ElementsDistribution();
+        HashMap<Character, Integer> expectedCharToCount = new HashMap<>();
+        expectedCharToCount.put('I',4);
+        expectedCharToCount.put('!',2);
+        expectedCharToCount.put('R',2);
+        expectedCharToCount.put('E',2);
+        expectedCharToCount.put('A',1);
+        expectedCharToCount.put('D',1);
+        expectedCharToCount.put('U',1);
+        expectedCharToCount.put('V',1);
+        expectedCharToCount.put('Y',1);
+        expectedCharToCount.put('N',1);
+        elementsDistribution.consoleOutput(expectedCharToCount);
+        String input = "Yurii Andrieiev!!";
+        HashMap<Character, Integer> charToCount = elementsDistribution.elementDistribution(input);
+        elementsDistribution.consoleOutput(charToCount);
+        assertEquals(expectedCharToCount, charToCount);
+    }
+
+    @Test
+    void shouldReturnSortedResultWhenWeGetStringInputThirdCase(){
```

Comment:
Very very bad naming. Your naming should express the purpose of particular test. In this, for example I would suggest -
elementDistribution_shouldDoRightOutput_whenInputHasManyWords